tools/image_processing/imagej2/imagej2_analyze_particles_binary.xml

```diff
@@ -97,32 +88,55 @@ $circularity_max
             <option value="no" selected="True">No</option>
             <option value="yes">Yes</option>
         </param>
+        <param name="export_rois" type="boolean" truevalue="true" falsevalue="false" checked="false" label="Export ROIs coordinates" help="For example to upload to OMERO" />
```

Comment:
I would change the label and help and do not use _ROIs_ but _particles outlines_.
A suggestion:

`<param name="export_rois" type="boolean" truevalue="true" falsevalue="false"
 checked="false" label="Export particles outlines coordinates"
 help="Exports particle coordinates as a tabular file, which can be imported as ROIs in OMERO." />`

Reply:
Very good suggestion. I incorporated it.